modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -2118,7 +2141,7 @@ public SegmentArchiveResult archiveSegment(long absIdx) throws StorageException
 
                 Files.move(dstTmpFile.toPath(), dstFile.toPath());
 
-                if (walCdcDir != null)
+                if (walCdcDir != null && !cdcForceDisable.getOrDefault(false))
```

Comment:
Let's rewrite a bit and warn user every time we skip link creation.

```
if (walCdcDir != null) {
    if (!cdcForceDisable.getOrDefault(false))
        Files.createLink(walCdcDir.toPath().resolve(dstFile.getName()), **dstFile.toPath());
    else 
        log.warning("Creation of segment CDC link skiped. `cdc.disabled` distributed property is false."); 
}
```

Reply:
Done.